Hammerspoon/setup.lua

```diff
@@ -1,10 +1,31 @@
 local modpath, frameworkspath, prettypath, fullpath, configdir, docstringspath, hasinitfile, autoload_extensions = ...
 
-package.path=configdir.."/?.lua"..";"..configdir.."/?/init.lua"..";"..configdir.."/Spoons/?.spoon/init.lua"..";"..package.path..";"..modpath.."/?.lua"..";"..modpath.."/?/init.lua"
-package.cpath=configdir.."/?.so"..";"..package.cpath..";"..frameworkspath.."/?.dylib"
+local userruntime = "~/.local/share/hammerspoon/site"
 
-print("-- package.path: "..package.path)
-print("-- package.cpath: "..package.cpath)
+local paths = {
+  configdir .. "/?.lua",
+  configdir .. "/?/init.lua",
+  configdir .. "/Spoons/?.spoon/init.lua",
+  package.path,
+  modpath .. "/?.lua",
+  modpath .. "/?/init.lua",
+  userruntime .. "/?.lua",
+  userruntime .. "/?/init.lua",
```

Comment:
Suggestion: add `userruntime .. "/Spoons/?.spoon/init.lua",` at end of list as first step in possibly migrating Spoon installation directory.